awsagentprovider/src/main/java/software/amazon/opentelemetry/javaagent/providers/exporter/otlp/aws/logs/OtlpAwsLogsExporter.java

```diff
@@ -0,0 +1,117 @@
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.opentelemetry.javaagent.providers.exporter.otlp.aws.logs;
+
+import io.opentelemetry.exporter.internal.otlp.logs.LogsRequestMarshaler;
+import io.opentelemetry.exporter.otlp.http.logs.OtlpHttpLogRecordExporter;
+import io.opentelemetry.exporter.otlp.http.logs.OtlpHttpLogRecordExporterBuilder;
+import io.opentelemetry.sdk.common.CompletableResultCode;
+import io.opentelemetry.sdk.common.export.MemoryMode;
+import io.opentelemetry.sdk.logs.data.LogRecordData;
+import io.opentelemetry.sdk.logs.export.LogRecordExporter;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.Collection;
+import java.util.StringJoiner;
+import javax.annotation.Nonnull;
+import software.amazon.opentelemetry.javaagent.providers.exporter.otlp.aws.common.BaseOtlpAwsExporter;
+
+/**
+ * This exporter extends the functionality of the OtlpAwsLogsExporter to allow logs to be exported
+ * to the CloudWatch Logs OTLP endpoint https://logs.[AWSRegion].amazonaws.com/v1/logs. Utilizes the
+ * AWSSDK library to sign and directly inject SigV4 Authentication to the exported request's
+ * headers. Also injects x-aws-log-group and x-aws-log-stream headers as per documentation: "<a
+ * href="https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-OTLPEndpoint.html">...</a>">
+ */
+public final class OtlpAwsLogsExporter extends BaseOtlpAwsExporter implements LogRecordExporter {
+  private final OtlpHttpLogRecordExporterBuilder parentExporterBuilder;
+  private final OtlpHttpLogRecordExporter parentExporter;
+
+  static OtlpAwsLogsExporter getDefault(String endpoint) {
+    return new OtlpAwsLogsExporter(endpoint);
+  }
+
+  static OtlpAwsLogsExporter create(OtlpHttpLogRecordExporter parent, String endpoint) {
+    return new OtlpAwsLogsExporter(parent, endpoint);
+  }
+
+  private OtlpAwsLogsExporter(String endpoint) {
+    this(null, endpoint);
+  }
+
+  private OtlpAwsLogsExporter(OtlpHttpLogRecordExporter parentExporter, String endpoint) {
+
+    super(endpoint);
+
+    if (parentExporter == null) {
+      this.parentExporterBuilder =
+          OtlpHttpLogRecordExporter.builder()
+              .setMemoryMode(MemoryMode.IMMUTABLE_DATA)
+              .setEndpoint(endpoint)
+              .setHeaders(this.authSupplier);
+      this.parentExporter = this.parentExporterBuilder.build();
+      return;
+    }
+
+    this.parentExporterBuilder =
+        parentExporter.toBuilder()
+            .setMemoryMode(MemoryMode.IMMUTABLE_DATA)
+            .setEndpoint(endpoint)
+            .setHeaders(this.authSupplier);
+
+    this.parentExporter = this.parentExporterBuilder.build();
```

Comment:
nit: Could avoid writing the `.build();` line twice
```suggestion
    if (parentExporter == null) {
      this.parentExporterBuilder =
          OtlpHttpLogRecordExporter.builder()
              .setMemoryMode(MemoryMode.IMMUTABLE_DATA)
              .setEndpoint(endpoint)
              .setHeaders(this.authSupplier);
    }
    else {
        this.parentExporterBuilder =
        parentExporter.toBuilder()
            .setMemoryMode(MemoryMode.IMMUTABLE_DATA)
            .setEndpoint(endpoint)
            .setHeaders(this.authSupplier);
    }
    
    this.parentExporter = this.parentExporterBuilder.build();
```